public/html/alerts/EditNotificationGroupViewModel.html

```diff
@@ -68,3 +69,11 @@ <h3>Recipients</h3>
         <a href="#" data-bind="click: $root.removeRecipient.bind($root)"><i class="fa fa-trash-o"></i></a>
     </li>
 </script>
+<script type="text/html" id="template-add-recipient-pagerduty">
```

Comment:
How is this used?

Reply:
I'll have to doublecheck. When I was implementing this, it seemed like it was required to get the controls working. If I add support for multiple pagerduty endpoints we'll need some sort of input control here.